src/main/java/calculator/Operation.java

```diff
@@ -0,0 +1,58 @@
+package calculator;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public enum Operation {
+    ADDITION("+") {
+        public int operate(int operand1, int operand2) {
+            return operand1 + operand2;
+        }
+    },
+    SUBTRACTION("-") {
+        public int operate(int operand1, int operand2) {
+            return operand1 - operand2;
+        }
+    },
+    MULTIPLICATION("*") {
+        public int operate(int operand1, int operand2) {
+            return operand1 * operand2;
+        }
+    },
+    DIVISION("/") {
+        public int operate(int operand1, int operand2) throws CalculateException {
+            if (operand2 == 0) {
+                throw new CalculateException();
+            }
+
+            return operand1 / operand2;
+        }
+    };
+
+    private final String OPERATION;
```

Comment:
멤버 변수 이름은 lowerCamelCase로 작성합니다. 아래의 Google Java Style Guide를 참고해 주세요.

> Non-constant field names (static or otherwise) are written in lowerCamelCase.